README.md

```diff
@@ -136,7 +135,7 @@ stacks:
 
 - `templates` - CloudFormation, SparkleFormation or CfnDsl templates.
 - `parameters` - Parameters as YAML files.
-- `secrets` - GPG encrypted secret files.
+- `secrets` - encrypted secret files.
```

Comment:
I don't think this parameter makes sense anymore without the GPG resolver... It's an unfortunate artefact of how it was originally written that we need to keep it. Perhaps we should just specify this option is only used for the GPG resolver.

Reply:
This is actually a list of directories used by StackMaster. The `secrets` directory is still used by the EJSON parameter resolver.